clpy/core/core.pyx

```diff
@@ -4382,7 +4386,7 @@ cpdef ndarray scan(ndarray a, ndarray out=None):
     if a.ndim != 1:
         raise TypeError("Input array should be 1D array.")
 
-    block_size = 256
+    block_size = 128
```

Comment:
Please leave a comment why this value is 128.

Reply:
On _vega_, `block_size = 256` didn't work due to `CL_DEVICE_MAX_WORK_GROUP_SIZE`.
Now I think this value should be determined through `GetDeviceInfo`.